features/font-variant-alternates.yml

```diff
@@ -3,3 +3,33 @@ description: The `font-variant-alternates` CSS property, along with the `@font-f
 spec: https://drafts.csswg.org/css-fonts-4/#font-variant-alternates-prop
 group: fonts
 caniuse: font-variant-alternates
+status:
+  compute_from: css.at-rules.font-feature-values
+compat_features:
+  - css.at-rules.font-feature-values
+  - css.at-rules.font-feature-values.annotation
+  - css.at-rules.font-feature-values.character-variant
+  - css.at-rules.font-feature-values.historical-forms
+  - css.at-rules.font-feature-values.ornaments
+  - css.at-rules.font-feature-values.styleset
+  - css.at-rules.font-feature-values.stylistic
+  - css.at-rules.font-feature-values.swash
+  - css.properties.font-variant-alternates
+  - css.properties.font-variant-alternates.annotation
+  - css.properties.font-variant-alternates.character_variant
+  - css.properties.font-variant-alternates.historical-forms
+  - css.properties.font-variant-alternates.normal
+  - css.properties.font-variant-alternates.ornaments
+  - css.properties.font-variant-alternates.styleset
+  - css.properties.font-variant-alternates.stylistic
+  - css.properties.font-variant-alternates.swash
+  - api.CSSFontFeatureValuesRule
+  - api.CSSFontFeatureValuesRule.annotation
+  - api.CSSFontFeatureValuesRule.characterVariant
+  - api.CSSFontFeatureValuesRule.fontFamily
+  # historicalForms is not present in BCD data
```

Comment:
It appears that historicalForms was [added to the spec](https://github.com/w3c/csswg-drafts/commit/5ad13d25e0c6ec21ac20fcb1fb9cfc039ec393c5) in Feb 2024, after the other values were [added to BCD](https://github.com/mdn/browser-compat-data/pull/19090) in March 2023.

Reply:
Do you know if anyone has actually implemented it yet? BCD won't track things without implementations. If it's implemented and not tracked, we can file an issue with the BCD and the collector tool to find out why it's not being detected. If it's merely unimplemented, we can ignore this for now and pick it up when it does appear.

Reply:
I haven't been able to track down any indication it has already had implementation, so I removed the comment here.